docs/changelog.md

```diff
@@ -317,6 +361,10 @@ description: |
   - Fixed path completions for `typst` not working in zsh
 
 - Tooling & Diagnostics
+  - Added [go to definition](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_definition)
```

Comment:
This already existed before 0.12 in the web app. Since we also haven't announced other web app features here that landed since 0.11, I've removed this.